modules/exploits/linux/http/saltstack_salt_wheel_async_rce.rb

```diff
@@ -0,0 +1,259 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+class MetasploitModule < Msf::Exploit::Remote
+
+  Rank = ExcellentRanking
+
+  prepend Msf::Exploit::Remote::AutoCheck
+  include Msf::Exploit::Remote::HttpClient
+  include Msf::Exploit::CmdStager
+  include Msf::Exploit::FileDropper
+
+  def initialize(info = {})
+    super(
+      update_info(
+        info,
+        'Name' => 'SaltStack Salt API Unauthenticated RCE',
```

Comment:
This can probably be a little more descriptive, since we have a few unauthed RCEs in Salt now.

Reply:
Absolutely, thanks!